style.css

```
@@ -0,0 +1,134 @@
+@import url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css");
+body {
+  font-family: "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI", "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #000000ef;
+}
+
+#header {
+  text-align: center;
+  font-size: 24px;
+  font-weight: bold;
+  color:white;
+  margin-bottom: 0px;
+}
+
+#main-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  margin: 20px auto;
+  width: 90%;
+  max-width: 700px;
+}
+
+#date-count-container {
+  color: white;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 90%;
+  max-width: 700px;
+  margin: 0 auto;
+}
+#todayDate{
+  font-size: 17px;
+}
+#count {
+  font-size: 15px;
+  display: flex;
+  gap: 15px;
+}
+
+#list {
+  list-style: none;
+}
+
+#list li {
+  font-size: 15px;
+  font-weight: bold;
+  background: #2c2c2c;
+  padding: 12px;
+  border-radius: 8px;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
+  margin-bottom: 10px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

입력창 텍스트가 너무 길어지면 줄바꿈이 되지 않고 칸을 뚫고 나가는 현상이 있습니다! 저는 한글이나 영어만 테스트해 보았다가 제출 후 숫자는 줄바꿈이 안 되어서 찾아보았는데 ```word-break: break-all```을 활용하면 모든 문자(숫자 포함)가 공백 상관 없이 줄바꿈이 잘 적용되더라구요!!
```suggestion
#list li {
  font-size: 15px;
  font-weight: bold;
  background: #2c2c2c;
  padding: 12px;
  border-radius: 8px;
  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
  margin-bottom: 10px;
  display: flex;
  justify-content: space-between;
  align-items: center;
  word-break: break-all;
}
```

---

너무 좋은 피드백입니다 ㅎㅎㅎ 영어, 한글, 숫자까지 테스트 해보다니요! 😍👍🏻👍🏻